src/reporters/standalone-test-case.reporter.ts

```diff
@@ -18,12 +20,17 @@ export class StandaloneTestCaseReporter implements CustomReporter {
     constructor() {
     }
 
+    public getCurrentSpec(): SpecResult | null {
+        return this.currentSpec;
+    }
+
     public jasmineStarted(suiteInfo: JasmineStartedInfo): void {
         (jasmine.getEnv() as any).testCaseReporter = this;
         console.log("");
     }
 
     public suiteStarted(suite: SuiteResult): void {
+        ScreenshotReporter.beforeSuite();
```

Comment:
I think this should be done at project level to reduce coupling between reporters. If we are using just this one, we shouldn't do anything regarding screenshots.

Reply:
Applied suggestion, removed from standaloneTestCaseReporter and handled at project level